### PR TITLE
Downgrade to stable dependencies and remove redundant settings

### PR DESCRIPTION
**Pull request by natemcmaster**
We downgraded all of our test projects to .NET Core 1.1 for now.

### CODE REVIEW

File: build/common.props [hunk 1]
```diff
@@ -1,12 +1,11 @@
 <Project>
+  <Import Project="dependencies.props" />
   <Import Project="..\version.props" />
 
   <PropertyGroup>
     <Product>Microsoft ASP.NET Core</Product>
     <RepositoryUrl>https://github.com/aspnet/Benchmarks</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <NetCoreAppImplicitPackageVersion>1.2.0-*</NetCoreAppImplicitPackageVersion>
```
Comment (mikeharder) [inline build/common.props]: Why was this removed?

File: experimental/HttpBenchmark/runtimeconfig.template.json [hunk 2]
```diff
@@ -1,5 +0,0 @@
-{
```
Comment (mikeharder) [inline experimental/HttpBenchmark/runtimeconfig.template.json]: Why was this removed?  Will Server GC still be enabled for this project?

File: src/Benchmarks/Benchmarks.csproj [hunk 3]
```diff
@@ -6,18 +6,10 @@
     <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
     <!-- TODO remove rid when https://github.com/dotnet/sdk/issues/396 is resolved -->
     <RuntimeIdentifier Condition="'$(TargetFramework)'!='netcoreapp1.1'">win7-x64</RuntimeIdentifier>
-    <PreserveCompilationContext>true</PreserveCompilationContext>
```
Comment (mikeharder) [inline src/Benchmarks/Benchmarks.csproj]: Why was this removed?

File: src/Benchmarks/Benchmarks.csproj [hunk 4]
```diff
@@ -6,18 +6,10 @@
...
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Update="appsettings.json" CopyToOutputDirectory="PreserveNewest" />
-    <Content Update="testCert.pfx" CopyToOutputDirectory="PreserveNewest" />
-    <Content Update="wwwroot\**\*" CopyToOutputDirectory="PreserveNewest" />
-    <Content Update="Views\**\*" CopyToOutputDirectory="PreserveNewest" />
-  </ItemGroup>
-
```
Comment (mikeharder) [inline src/Benchmarks/Benchmarks.csproj]: Why were these removed?

File: build/common.props (hunk 1 above)
Comment (natemcmaster) [inline build/common.props]: Property was a temporary fix. SDK doesn't use it anymore.

File: experimental/HttpBenchmark/runtimeconfig.template.json (hunk 2 above)
Comment (natemcmaster) [inline experimental/HttpBenchmark/runtimeconfig.template.json]: Replaced by the MSBuild setting `<ServerGarbageCollection>true</ServerGarbageCollection>`

File: src/Benchmarks/Benchmarks.csproj (hunk 3 above)
Comment (natemcmaster) [inline src/Benchmarks/Benchmarks.csproj]: This is the default setting for Microsoft.NET.Sdk.Web projects.

File: src/Benchmarks/Benchmarks.csproj (hunk 4 above)
Comment (natemcmaster) [inline src/Benchmarks/Benchmarks.csproj]: They're unnecessary to copy to the build output folder.
